example_experiments/reading_comprehension/gareader_who_did_what.json

```diff
@@ -1,11 +1,15 @@
 {
     "embeddings": {
       "words": {
-        "pretrained_file": "/efs/data/dlfa/glove/glove.6B.100d.txt.gz",
+        "pretrained_file": "/net/efs/data/dlfa/glove/glove.6B.100d.txt.gz",
         "fine_tune": false,
-        "project": false
+        "project": false,
+          "droppout": 0.0
       },
-      "characters": {"dimension": 16}
+      "characters": {
+          "dimension": 16,
```

Comment:
Two space indents, I think.

Honestly, I don't much care which it is, but I noticed it because it's not consistent in the file.  I just checked other files, and the spacing isn't consistent across files.  It's really not worth standardizing all of them, but at least within a file, the indentation should be consistent.